### DISCUSSION TITLE
Welcome to nextjs-starter-medusa Discussions!

### DISCUSSION THREAD

**Discussion by VariableVic** [medusajs/nextjs-starter-medusa#241] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by mohdziyadc** [comment]
Hey, my name's Ziyad. I'm currently using medusa for a project I'm involved. For the last 15 days, I have been building a custom storefront with the starter template. Since you guys have upgraded the starter template and made it better, I'd like to upgrade my existing storefront. But I'm finding it really hard to do so. It would be really helpful if you guys could provide me with an upgrade guide to resolve this matter.

**Comment by iamagenus** [comment]
I would love to use this for an e commerce store as well, but I am having trouble grasping how to do some things. A little help on creating a  hero banner or some basic e commerce footer components would be lovely. If anyone can point me to some references I would be grateful. Really nice storefront overall.

**Comment by acoronasz** [comment]
Hi, where can we get working version of update password and logout for the starters torefront. This seems to be working fine on live demo site but when setting it up from repository the password update is not available and logout fails to send to login page. Is there a branch with these changes, or do we have to make these changes?

Thanks
